pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -216,7 +225,7 @@ func (qjm *XController) allocatableCapacity() *clusterstateapi.Resource {
 			}
 		}
 	}
-	klog.Infof("[allocatableCapacity] The available capacity to dispatch appwrapper is %v and time took to calculate is %v", capacity, time.Since(startTime))
+	klog.V(4).Infof("[allocatableCapacity] The available capacity to dispatch appwrapper is %v and time took to calculate is %v", capacity, time.Since(startTime))
```

Comment:
I think we should have this log line at a higher logging level, meaning at 2 or 3. This log line tells us what is the cluster capacity when an AW is considered to be dispatched.